src/site/markdown/server/manage-webadmin.md

```diff
@@ -3048,6 +3049,40 @@ Response codes:
  - 204: Success (No content)
  - 400: Invalid request
  - 404: The mail queue does not exist
+ 
+### RabbitMQ republishing a mail queue from cassandra
+
+```
+curl -XPOST http://ip:port/mailQueues/{mailQueueName}?action=RepublishNotProcessedMails&olderThan=1d
```

Comment:
Should be

```
curl -XPOST 'http://ip:port/mailQueues/{mailQueueName}?action=RepublishNotProcessedMails&olderThan=1d'
```

Some production incidents happened because of `&` being bash interpreted instead of curl interpreted.

We of course need to correct this for all endpoints, but in a separated PR.